plugin-server/src/main/utils.ts

```diff
@@ -20,18 +27,27 @@ export async function runInstrumentedFunction<T>({
     func,
     statsKey,
     teamId,
+    logToConsole = false,
```

Comment:
this is a confusing env name as timeoutGuard also logs to console, you'd expect it not to do that. So maybe `logExecutionTimes`

Reply:
Fair point 👍